helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/collectiontransformer/CharSequenceListView.java

```diff
@@ -60,4 +60,9 @@ public boolean addAll(int index, java.util.Collection<? extends CharSequence> c)
   public boolean remove(Object o) {
     return utf8List.remove(new Utf8(o.toString()));
   }
+
+  @Override
+  public void clear() {
+    utf8List.clear();
+  }
 }
```

Comment:
Can you add doc why these method need to be override, the thing you have the PR description.